AntistasiOfficial.Altis/Municion/NATOCrate.sqf

```diff
@@ -23,146 +23,47 @@ clearBackpackCargoGlobal _crate;
-
-	// LMG
-	_crate addWeaponCargoGlobal [selectRandom _lmgs, 1];
-	_crate addMagazineCargoGlobal [bluLMGAmmo select 2, (2+(floor random 3))];
-
-	// shotgun/SMG
-	_crate addWeaponCargoGlobal [bluSmallWpn select 0, 1];
-	_crate addMagazineCargoGlobal [selectRandom _smAmmo, (4+(floor random 6))];
-
-	// sidearm
-	_crate addWeaponCargoGlobal [bluSmallWpn select 1, 1];
-	_crate addMagazineCargoGlobal [bluSmallAmmo select 2, (6+(floor random 3))];
-
-	// AT launcher
-	_crate addWeaponCargoGlobal [bluAT select 0, 1];
-	_crate addMagazineCargoGlobal [selectRandom bluATMissile, (3+(floor random 3))];
-
-	// AA launcher
-	_crate addWeaponCargoGlobal [bluAA select 0, 1];
-	_crate addMagazineCargoGlobal [selectRandom bluAAMissile, (3+(floor random 3))];
-
-	// scopes
-	_crate addItemCargoGlobal [selectRandom bluScopes, 2];
-
-	// attachment
-	_crate addItemCargoGlobal [selectRandom bluAttachments, 4];
-}
-else {
-	if (_NATOSupp > 60) then {
-		// sniper rifle
-		_crate addWeaponCargoGlobal [bluSNPR select 0, 1];
-		_crate addMagazineCargoGlobal [bluSNPRAmmo select 1, (2+(floor random 4))];
-
-		// rifle with 40mm
-		_crate addWeaponCargoGlobal [selectRandom bluGL, 2];
-		_crate addMagazineCargoGlobal [selectRandom bluRifleAmmo, 2*(3+(floor random 4))];
-		_crate addMagazineCargoGlobal [blu40mm select 0, 2*(2+(floor random 4))];
-		_crate addMagazineCargoGlobal [selectRandom blu40mm, 2*(1+(floor random 3))];
-
-		// LMG
-		_crate addWeaponCargoGlobal [bluLMG select 0, 1];
-		_crate addMagazineCargoGlobal [selectRandom _lmgAmmo, (2+(floor random 3))];
-
-		// shotgun/SMG
-		_crate addWeaponCargoGlobal [bluSmallWpn select 0, 1];
-		_crate addMagazineCargoGlobal [bluSmallAmmo select 1, (4+(floor random 2))];
-
-		// AT launcher
-		_crate addWeaponCargoGlobal [bluAT select 0, 1];
-		_crate addMagazineCargoGlobal [selectRandom bluATMissile, (3+(floor random 3))];
-
-		// scopes
-		_crate addItemCargoGlobal [selectRandom bluScopes, 1];
-
-		// attachment
-		_crate addItemCargoGlobal [selectRandom bluAttachments, 3];
+//Standard Equipment (good for 5 people)
+	//										Weapon						
+	_crate addWeaponCargoGlobal 		[bluSmallWpn 	select 0,	5	];	
+	_crate addMagazineCargoGlobal 		[bluSmallAmmo 	select 0,	25	];
+	_crate addWeaponCargoGlobal 		[bluSmallWpn 	select 1,	5	];
+	_crate addMagazineCargoGlobal 		[bluSmallAmmo 	select 1,	25	];
+	_crate addItemCargoGlobal			["SmokeShellRed"		,	25	];
+	_crate addItemCargoGlobal			["SmokeShellGreen"		,	25	];
+	_crate addItemCargoGlobal				
+	_crate addItemCargoGlobal			[bluAttachments select 0,	5	];	//flashlight
+
+	//										Medical
+	if !(activeACEMedical) then( 
+		_crate addItemCargoGlobal		["FirstAidKit"			,	25	];
+		)
+		else( 
+		_crate addItemCargoGlobal		["ACE_epinephrine"		,	10	];		
+		_crate addItemCargoGlobal		["ACE_morphine"			,	25	];
+		_crate addItemCargoGlobal		["ACE_fieldDressing"	,	25	];
+		_crate addItemCargoGlobal		["ACE_bloodIV"			,	40	];
+		)
```

Comment:
First of all, `{}` are missed, then ending `;`
These items are already unlimited in Arsenal, and actually i don't see sense in limiting med supplies.

Reply:
Thanks for corrections
Because the resupply is meant to be delivered in the field where you don't have unlimited stuff